tests/FakeItEasy.Tests.TestHelpers/UsingCultureAttribute.cs

```diff
@@ -23,8 +23,17 @@ public UsingCultureAttribute(string cultureName)
 
         public override void After(MethodInfo methodUnderTest)
         {
-            CultureInfo.CurrentCulture = this.originalCulture;
-            CultureInfo.CurrentUICulture = this.originalUiCulture;
+            if (this.originalCulture is CultureInfo originalCulture)
+            {
+                CultureInfo.CurrentCulture = originalCulture;
+                this.originalCulture = null;
+            }
+
+            if (this.originalUiCulture is CultureInfo originalUiCulture)
+            {
+                CultureInfo.CurrentUICulture = originalUiCulture;
+                this.originalUiCulture = null;
+            }

```

Comment:
We knew After would never be called before Before, but the compiler didn't...

Reply:
Ugh. I mean, it's right, but ugh.